chronos/transports/stream.nim

```diff
@@ -1594,7 +1657,7 @@ proc read*(transp: StreamTransport, n = -1): Future[seq[byte]] {.async.} =
     if transp.offset > 0:
       let s = len(result)
       let o = s + transp.offset
-      if n < 0:
+      if n <= 0:
```

Comment:
Why the need for this change? This will give both for `n = -1` and `n = 0` read all bytes right?
I can understand that n=0 does not make much sense else, but perhaps update the documentation also then.

Reply:
Fixed documentation.